engine/gamesys/src/gamesys/scripts/box2d/script_box2d_body.cpp

```diff
@@ -0,0 +1,971 @@
+ * @return enabled [type: bool] true if the body is allowed to sleep
+ */
+
+/*# Set the sleep state of the body. A sleeping body has very low CPU cost.
+ * @name body:set_awake
+ * @param enable [type: bool] flag set to false to put body to sleep, true to wake it.
+ */
+
+/*# Get the sleeping state of this body.
+ * @name body:is_awake
+ * @return enabled [type: bool] true if the body is awake, false if it's sleeping.
+ */
+
+/*# Set the active state of the body
+ * Set the active state of the body. An inactive body is not
+ * simulated and cannot be collided with or woken up.
+ * If you pass a flag of true, all fixtures will be added to the
+ * broad-phase.
+ * If you pass a flag of false, all fixtures will be removed from
+ * the broad-phase and all contacts will be destroyed.
+ * Fixtures and joints are otherwise unaffected. You may continue
+ * to create/destroy fixtures and joints on inactive bodies.
+ * Fixtures on an inactive body are implicitly inactive and will
+ * not participate in collisions, ray-casts, or queries.
+ * Joints connected to an inactive body are implicitly inactive.
+ * An inactive body is still owned by a b2World object and remains
+ * in the body list.
+ *
+ * @name body:set_active
+ * @param enable [type: bool] true if the body should be active
+ */
+
+/*# Get the active state of the body.
+ * @name body:is_active
+ * @return enabled [type: bool] is the body active
+ */
+
+
+/*# Set this body to have fixed rotation. This causes the mass to be reset.
+ * @name body:set_fixed_rotation
+ * @param enable [type: bool] true if the rotation should be fixed
+ */
+
+/*# Does this body have fixed rotation?
+ * @name body:is_fixed_rotation
+ * @return enabled [type: bool] is the rotation fixed
+ */
+
+/** Get the list of all fixtures attached to this body.
+ * @name body:get_fixture_list
+ * @return edge [type: b2Fixture] the first fixture
+ */
+
+/** Get the list of all joints attached to this body.
+ * @name body:get_joint_list
+ * @return edge [type: b2JointEdge] the first joint
+ */
+
+/** Get the list of all contacts attached to this body.
+ * @name body:get_contact_list
+ * @return edge [type: b2ContactEdge] the first edge
+ */
+
+/*# Get the next body in the world's body list.
+ * @name body:get_next
+ * @return body [type: b2Body] the next body
+ */
+
+/*# Get the user data pointer that was provided in the body definition.
+ * @name body:get_user_data
+ * @return id [type: hash] the game object id this body is connected to
+ */
+
+/** Set the user data. Use this to store your application specific data.
+ * @name body:set_user_data
+ * @param id [type: hash] the game object id
+ */
+
+/*# Get the parent world of this body.
+ * @name body:get_world
+ * @return world [type: b2World]
+ */
+
+/*# Print the body representation to the log output
+ * @name body:dump
+ */
+
+/** Get the total force currently applied on this object
+ * @name body:get_force
+ * @note Defold Specific
+ * @return force [type: vmath.vector3]
```

Comment:
it seems like this function is already implemented but it's commented (as I understand `/**` - comment out function, and `/*#` - added function?)

Reply:
Correct.